x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/edit_policy.js

```diff
@@ -63,15 +63,30 @@ class EditPolicyUi extends Component {
       setSelectedPolicy(selectedPolicy);
     }
   };
+  getPolicyNameFromUri() {
+    // This method is necessary because react-router mangles path params that are encoded
+    // for example: http://localhost:5601/zpf/app/kibana#/management/elasticsearch/index_lifecycle_management/policies/edit/asdfasdFW%23@@$%25%23%5E%25%3F%23%25%5E&%3F%23%25%5E&%3F%23%25%5E&%3F%23%25%5E&%23%25%5E*%3F
+    // policyName winds up getting URI encoded twice to prevent an issue where the back button does not work
+    const { hash } = window.location;
+    if (!hash.includes('?')) {
+      return null;
+    }
+    const query = hash.split('?').pop();
+    const { policyName } = queryString.parse(query);
```

Comment:
It looks like we're looking for the policy name as part of the query string, but in the comment's example on line 68, it's part of the path param. Is the comment outdated?

Reply:
yeah removed this

Reply:
removed